<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Neeru" failures="1" tests="4" name="Test" time="56.861" errors="0" timestamp="2024-04-19T01:00:37 IST">
  <testcase classname="my_profile.add_wallet_points" name="verify_hardcoded_amount_available" time="1.095"/>
  <testcase classname="my_profile.add_wallet_points" name="wallet_balance_verification" time="0.033"/>
  <testcase classname="my_profile.add_wallet_points" name="error_amount_verification" time="7.317"/>
  <testcase classname="my_profile.add_wallet_points" name="error_rajorpay_verification" time="4.146">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Alert message is mismatched by cancelling rajor pay payment  expected [Are you sure, you want to close the for?] but found [Are you sure, you want to close the form?]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Alert message is mismatched by cancelling rajor pay payment  expected [Are you sure, you want to close the for?] but found [Are you sure, you want to close the form?]
at my_profile.add_wallet_points.error_rajorpay_verification(add_wallet_points.java:193)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- error_rajorpay_verification -->
</testsuite> <!-- Test -->
